crates/anvil/tests/it/sign.rs

```diff
@@ -88,7 +92,7 @@ async fn can_sign_typed_data() {
         .unwrap();
     assert_eq!(
       signature,
-      "0x6ea8bb309a3401225701f3565e32519f94a0ea91a5910ce9229fe488e773584c0390416a2190d9560219dab757ecca2029e63fa9d1c2aebf676cc25b9f03126a1b".to_string()
+      "0x6ea8bb309a3401225701f3565e32519f94a0ea91a5910ce9229fe488e773584c0390416a2190d9560219dab757ecca2029e63fa9d1c2aebf676cc25b9f03126a01".to_string()
```

Comment:
why are these different?

Reply:
re, the difference cc @onbjerg, i believe it's due to this: https://github.com/gakonst/ethers-rs/blob/master/ethers-signers/src/wallet/mod.rs#L152. In ethers, v is made to be 27/28 when signing a hash and stored as such, which is then the last byte shown on the signed data here. The way we do it we just keep it as a boolean, and append that.